config/go.mod

```diff
@@ -5,6 +5,11 @@ go 1.22.0
 require (
 	github.com/prometheus/client_golang v1.20.5
 	github.com/stretchr/testify v1.10.0
+	go.opentelemetry.io/contrib/propagators/autoprop v0.59.0
+	go.opentelemetry.io/contrib/propagators/aws v1.34.0
```

Comment:
Is the package missing a replace to introduce the configuration of the local package?

Other packages are similar.

Reply:
I ran crosslink which brought more replace statements. Is that what you had in mind?

Reply:
Although it may be a bit cumbersome. However, I'm thinking that if we don't do this, it means that the dependencies of the current package will always be the previous release version.

If we make some incompatible adjustments to the dependencies in the current unreleased version, our unit tests may not be able to detect them. The problem won't be found until after the official release.

Reply:
@flc1125, I can see replace for all these modules at the end of the file.

Reply:
> I did see it. This is a comment, not a discussion that must be modified. 

I do not think it was clear. 

> Therefore, there is no "Resolved" button.

I do see a "Resolve conversation" button. Resolving now :wink: